style.css

```
@@ -3,31 +3,49 @@
 }
 
 body {
-  background: #222;
-  margin: 2%;
+  background: rgb(20, 20, 20);
+  min-width: 150px; /*Support for smaller device that are skinny and tall. Keep me updated if this isnt enough.*/
```

Let's go ahead and upgrade this to 60em, in my testing that works a lot better. I'd prefer this to be em as it guarantees we can fit a certain amount of characters.
```suggestion
  min-width: 60em; /*Support for smaller device that are skinny and tall. Keep me updated if this isnt enough.*/
```